src/components/PaginatedTable/Customize/CellView.js

```
@@ -0,0 +1,287 @@
+        )
+      }
+    </div>
+  );
+};
+
+const ArmsCohortList = (props) => {
+  const { dataField } = props;
+  const value = props[dataField];
+  return value
+    .split('#').map((desc) => (desc === '' ? ''
+      : <li style={{ listStyleType: 'none' }}>{desc}</li>));
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)